Exiled.API/Features/Player.cs

```diff
@@ -1251,6 +1256,38 @@ public static Player Get(string args)
         /// <returns>A boolean indicating whether or not a player was found.</returns>
         public static bool TryGet(string args, out Player player) => (player = Get(args)) is not null;
 
+        /// <summary>
+        /// Adds a player's UserId to the list of reserved slots.
+        /// </summary>
+        /// <remarks>This method does not permanently give a user a reserved slot. The slot will be removed if the reserved slots are reloaded.</remarks>
+        /// <param name="userId">The UserId of the player to add.</param>
+        /// <returns><see langword="true"/> if the slot was successfully added, or <see langword="false"/> if the provided UserId already has a reserved slot.</returns>
+        /// <seealso cref="AddReservedSlot(Player)"/>
+        public static bool AddReservedSlot(string userId)
```

Comment:
Should we name this method and the one below `GiveReservedSlot`?

Reply:
We could just call the one below `GiveReservedSlot`, since it's not static, this one is actually fine.